configstack/running_module.go

```diff
@@ -104,9 +105,27 @@ func crossLinkDependencies(modules map[string]*runningModule, dependencyOrder De
 		}
 	}
 
+	removeFlagExcluded(modules)
+
 	return modules, nil
 }
 
+// Removes all the modules from the map, which have been flagged as excluded in flagExcludedDirs()
+func removeFlagExcluded(modules map[string]*runningModule) {
```

Comment:
Instead mutating the existing map, I'd just create a new one that only contains the non-excluded modules.

Reply:
Adjusted this accordingly, however the original solution was probably more efficient and had better readability.